src/app/modules/sgid/sgid.controller.ts

```diff
@@ -0,0 +1,82 @@
+import { ParamsDictionary, RequestHandler } from 'express-serve-static-core'
```

Comment:
we've removed `express-serve-static-core` from the app, use `ControllerHandler` from `'core/core.types'` instead? :P